lib/components/CourseView.jsx

```diff
@@ -59,6 +60,24 @@ const recipeState = recipeCheckables => {
 };
 
 const btnClassName = 'checkable-button btn btn-primary btn-nordic';
+const deprecated = process.platform === 'win32';
+const deprecatedClassName = deprecated ? 'deprecated' : '';
```

Comment:
Since the `deprecatedClassName` is only used in https://github.com/NordicSemiconductor/pc-nrfconnect-gettingstarted/pull/22/files#diff-d186825f4ac178d5d3740b8ecb43ffa6R118-R120 if `deprecated` is true, you don't need a ternary for `deprecatedClassName`, just set it fixed to `'deprecated'` (and maybe inline it then).

Also: I think the name `'deprecated-watermark'` would be a better class name.